.github/workflows/tools.yml

```
@@ -21,12 +21,11 @@ jobs:
             subsystem: tools
             label: tools
             run: |
-              NEW_VERSION=$(npm view eslint dist-tags.latest)
-              CURRENT_VERSION=$(node -p "require('./tools/node_modules/eslint/package.json').version")
-              if [ "$NEW_VERSION" != "$CURRENT_VERSION" ]; then
-                echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
-                ./tools/dep_updaters/update-eslint.sh
-              fi
+              ./tools/dep_updaters/update-eslint.sh > temp-output
+              cat temp-output
+              MAYBE_VERSION=$(tail -n1 temp-output)
+              if grep -q "NEW_VERSION" <<< "$MAYBE_VERSION"; then $MAYBE_VERSION >> $GITHUB_ENV; fi
```

Here's how I would do it, not sure if it's better but maybe it is easier to follow? I'll let you be the judge and use the syntax you prefer.
```suggestion
              echo "$MAYBE_VERSION" | grep -q "NEW_VERSION=" && echo "$MAYBE_VERSION" >> $GITHUB_ENV
```

---

I opted for the second one ^^ This PR definitely helped me to improve my scripting skills. Thanks for all your comments 🙌🏼